pulsar-functions/runtime/src/main/java/org/apache/pulsar/functions/runtime/RuntimeSpawner.java

```diff
@@ -130,9 +130,6 @@ public void close() {
             runtime.stop();
             runtime = null;
         }
-        if (runtimeFactory != null) {
```

Comment:
Please see my comments for this on: #1988

Reply:
@rdhabalia shall we rebase this PR since #1988 is merged?